util/envDynawoAlgorithms.sh

```diff
@@ -353,7 +354,7 @@ config_dynawo_algorithms() {
     -DLIBZIP_HOME=$DYNAWO_LIBZIP_HOME \
     -DCMAKE_MODULE_PATH=$DYNAWO_HOME/share/cmake \
     -DDYNAWO_PYTHON_COMMAND="$DYNAWO_ALGORITHMS_PYTHON_COMMAND" \
-    -G "Unix Makefiles" \
+    -G $DYNAWO_CMAKE_GENERATOR \
```

Comment:
"$DYNAWO_CMAKE_GENERATOR"

Reply:
quotes are already in the variable when set

Reply:
yes but you need to add them here to to avoid the issue with the space

Reply:
When put it that way, there is no problem with "Unix Makefiles"

Reply:
Problem or not it is mandatory in bash to quotes variables and a bad practice not to quote them